### DISCUSSION TITLE
[Question]: Inconsistency in docs about using existing project icons

### DISCUSSION THREAD

**Discussion by KiGamji** [danny-avila/LibreChat#5095] [Q&A]
### What is your question?

The documentation part [here](https://www.[librechat.ai/docs/configuration/librechat_yaml/object_structure/custom_endpoint](https://www.librechat.ai/docs/configuration/librechat_yaml/object_structure/custom_endpoint)) clearly says:
> If you want to use existing project icons, define the endpoint name as one of the main endpoints (case-sensitive):
> “openAI” | “azureOpenAI” | “google” | “anthropic” | “assistants” | “gptPlugins”

However, when trying to do so, I am getting this error:
```
error: Invalid custom config file at /app/librechat.yaml:
{
  "issues": [
    {
      "code": "custom",
      "message": "Value cannot be one of the default endpoint (EModelEndpoint) values: azureOpenAI, openAI, bingAI, chatGPTBrowser, google, gptPlugins, anthropic, assistants, azureAssistants, agents, custom, bedrock",
      "path": [
        "endpoints",
        "custom",
        0,
        "name"
      ]
    }
  ],
  "name": "ZodError"
}
```

It would be very nice to keep this feature, if possible. Thanks in advance!

### More Details

-

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by danny-avila** [comment]
this is a misunderstanding, the part you are quoting is referring to the value for `endpoints.custom.iconURL` not `endpoints.custom.name`

**Comment by danny-avila** [comment]
I see that the docs should be fixed, so I'm moving this to the docs repo.

**Reply by danny-avila** [reply]
https://github.com/LibreChat-AI/librechat.ai/issues/187
